.ci/jenkins/Jenkinsfile.release

```diff
@@ -306,31 +306,27 @@ pipeline {
              }
          }
         // send email for Sanity Checks
-        stage ('Email send with BUILD result') {
-            when{
-                expression { reBuild == 'YES' }
-            }
-            steps {
-                emailext body: "Build of community ${kieVersion} was:  ${currentBuild.currentResult} \n" +
-                    " \n" +
-                    "Failed tests: ${BUILD_URL}/testReport \n" +
-                    " \n" +
-                    " \n" +
-                    "The artifacts are available here \n" +
-                    " \n" +
-                    "business-central artifacts: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/business-central/${kieVersion}/ \n" +
-                    "business-central-webapp: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/business-central-webapp/${kieVersion}/ \n" +
-                    "business-monitoring-webapp: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/business-monitoring-webapp/${kieVersion}/ \n" +
-                    " \n" +
-                    "Please download for sanity checks: jbpm-server-distribution.zip: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/jbpm-server-distribution/${kieVersion}/ \n" +
-                    " \n" +
-                    " \n" +
-                    "Please download the needed binaries, fill in your assigned test scenarios and check the failing tests \n" +
-                    "sanity checks: https://docs.google.com/spreadsheets/d/1jPtRilvcOji__qN0QmVoXw6KSi4Nkq8Nz_coKIVfX6A/edit#gid=167259416 \n" +
-                    " \n" +
-                    "In case Sanity Checks were already done and this kind of mail arrives the second time, please verify if the bugs reported in Sanity Checks are fixed now. \n" +
-                    " \n" +
-                    "KIE version: ${kieVersion}", subject: "community-release-${baseBranch} ${kieVersion} status and artefacts for sanity checks", to: "bsig@redhat.com"
+        stage('send email with build result'){
+            steps{
```

Comment:
The same here
```
steps{
                emailext body: """
The satus of community release ${kieVersion} was ${currentBuild.currentResult}

Failed tests: ${BUILD_URL}testReport

The artifacts are available here

business-central artifacts: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/business-central/${kieVersion}/ 
business-central-webapp: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/business-central-webapp/${kieVersion}/ 
business-monitoring-webapp: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/business-monitoring-webapp/${kieVersion}/ 

Please download for sanity checks: jbpm-server-distribution.zip: https://repository.jboss.org/nexus/content/groups/kie-group/org/kie/jbpm-server-distribution/${kieVersion}/

Please download the needed binaries, fill in your assigned test scenarios and check the failing tests
sanity checks: https://docs.google.com/spreadsheets/d/1jPtRilvcOji__qN0QmVoXw6KSi4Nkq8Nz_coKIVfX6A/edit#gid=167259416

In case Sanity Checks were already done and this kind of mail arrives the second time, please verify if the bugs reported in Sanity Checks are fixed now.

KIE version: ${kieVersion}""", subject: "community-release for ${kieVersion}", to: 'bsig@redhat.com'
            }
```

Reply:
DONE